tests/test_interact.py

```diff
@@ -62,10 +63,12 @@ def test_interact_escape_None(self):
         p.sendcontrol(']')
         p.expect('29<STOP>')
         p.send('\x00')
-        if not os.environ.get('TRAVIS', None):
-            # on Travis-CI, we sometimes miss trailing stdout from the
-            # chain of child processes, not entirely sure why. So this
-            # is skipped on such systems.
+
+        if not platform.system() == 'Linux':
```

Comment:
@jquast I'd like to get this test question sorted out, merge this and cut a new release (I guess there's enough changes to call it 4.1). I'd like to get the fix from #306 out soon, so if we need longer to work this out, I may cut a new release without it.